jenkins/pyinstaller/v2/requirements.txt

```diff
@@ -5,4 +5,4 @@ argparse==1.4.0
 dataclasses==0.7
 python-consul==1.1.0
 pathlib==1.0.1
-pytest==5.4.2
+pytest==5.4.2
```

Comment:
This does look unrelated to merge request.

Reply:
It is no change. Vscode may have done auto correction by crop black space  or line